### ISSUE TITLE
open with function always disabled

### ISSUE THREAD

**Issue by penance316** [mucommander/mucommander#96]
Mac OSX El Capitan 10.11.6
muCommander 0.9.1

the "Open With" option is always disabled no matter what file type I am right clicking on.

Steps:
1. Navigate to folder
2. right click on any file (in my example test.csv)
3. note that 'open with' option is disabled and un-clickable

**Comment by ahadas** [comment]
Hi @penance316 ,
This is not really a bug. The "Open with" menu only displays [custom commands](https://github.com/mucommander/mucommander/wiki/CustomCommands). You probably did not set ones.

**Comment by slsu0424** [comment]
@penance316 - Were you able to get this to work?  I followed the instructions to set up commands.xml file, yet the "Open With" option is still grayed out for me.  

FYI - I have the same specs as you - OSX El Capitan, muCommander 0.9.1

**Comment by penance316** [comment]
@slsu0424 Yes mine is working now. 
I had to create a commands.xml file in the ~/Library/Preferences/muCommander folder. I used the sample custom commands as mentioned in the  readme (https://github.com/mucommander/mucommander/wiki/Custom-Commands#custom-commands-format)

**Comment by slsu0424** [comment]
@penance316 - That's interesting.  I've followed the exact same steps and still the "Open With" is grayed out for me.

@ahadas - Is there something I am missing to get the "Open With" option to not be grayed out?  I've followed the instructions listed out here: https://github.com/mucommander/mucommander/wiki/Custom-Commands

A sample of my commands.xml file looks as follows:

`<?xml version="1.0" encoding="UTF-8"?>
<commands>
    <command alias="Preview" value="open -a Preview $f"/>
</commands>
`

Appreciate the help in advance.
